test/integration/backward_compatible/ClientHotRestartEventTest.js

```diff
@@ -82,6 +82,7 @@ describe('ClientHotRestartEventTest', function () {
     });
 
     it('should receive membership events when the member is restarted on same address', async function () {
+        TestUtil.markClientVersionAtLeast(this, '4.2');
```

Comment:
We should probably update `afterEach` as well, since the client won't be constructed

Reply:
good catch 👍🏻